confidant/webhook.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+import logging
+
+from confidant.app import app
+
+
+def send_event(event_type, services, credentials):
```

Comment:
Can you change `credentials` to `credential_ids` to clarify that folks should send ids and never credentials into this :).

Also, can you add a function doc string talking about the args and the expected input? We really want to make sure no one ever accidentally calls this with actual credentials, and only ever a list of IDs.